README.md

```diff
@@ -42,6 +42,20 @@ Alternatively, you can test specific parts of the app:
 
 `yarn test:(server|types|app)`
 
+### Development
+
+Bypass Chrome's security restrictions for local development:
+
+1. Add the following entry to your hosts file: `127.0.0.1 webapp.zinfra.io`
```

Comment:
```suggestion
1. Add the following entry to your hosts file: `127.0.0.1 local.zinfra.io`
```
so far we used all used `local.zinfra.io`

Reply:
I think it would be good to be in sync here in order to simplify communication within the team.